src/test/java/pl/bajerska/befindyourmeal/BeFindYourMealApplicationTests.java

```diff
@@ -0,0 +1,13 @@
+package pl.bajerska.befindyourmeal;
```

Comment:
This can be removed as well. Empty test file looks bad. Or... some tests can be added. :)